charts/infra/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: infra
 description: A Helm chart for Infra connector. For Infra server charts, use infra-server.
 type: application
 version: 0.1.1
```

Comment:
`version` also needs to be bumped if you're bumping the `appVersion`

Reply:
Great catch, thanks!